Detections/MultipleDataSources/IridiumIOCs.yaml

```diff
@@ -0,0 +1,121 @@
+id: 7ee72a9e-2e54-459c-bc8a-8c08a6532a63
+name: Known IRIDIUM IP
+description: |
+  'IRIDIUM command and control IP. 
+   Matches  IP IOCs related to the IRIDIUM activity group with CommonSecurityLog, OfficeActivity, DnsEvents, VMConnection and SecurityEvents dataTypes.'
+severity: High
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+     - OfficeActivity
+  - connectorId: DNS
+    dataTypes:
+      - DnsEvents
+  - connectorId: AzureMonitor(VMInsights)
+    dataTypes:
+      - VMConnection
+  - connectorId: CiscoASA
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: PaloAltoNetworks
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs
+  - connectorId: AzureMonitor(WireData)
+    dataTypes:
+      - WireData
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+query: | 
+ let timeframe = 1d;
```

Comment:
Please make sure there is 2 spaces in front of the query lines.

Reply:
Done. Thanks!!